test/MathUtils.test.cpp

```diff
@@ -218,3 +218,31 @@ TEST(MathUtils, isRectInRectClass)
 	EXPECT_EQ(false, NAS2D::isRectInRect({5, 5, 1, 1}, {3, 3, 1, 1}));
 	EXPECT_EQ(false, NAS2D::isRectInRect({5, 5, 1, 1}, {7, 3, 1, 1}));
 }
+
+TEST(MathUtils, mapDomainToRange)
+{
+	//I know the temperature conversions have established formulas, but that's not what I'm testing.
+	//Given proper input I should get the same value as if I used the derived formulas.
+
+	//(v - minD) * (maxR - minR) / (maxD - minD) + minR;
+	//(F - 32.0f) * (5.0f / 9.0f) + 0.0f;
+	//(C - 0.0f) * (9.0f / 5.0f) + 32.0f;
+
+	//Fahrenheit to Celcius
```

Comment:
Very minor point, though most comments in this code base have a space after the `//`
```
// Fahrenheit to Celcius
```

Same applies to the other comments in this file.